frontend/express/public/core/user-management/templates/user-management.html

```diff
@@ -10,20 +10,16 @@
 							</div>
 					</template>
 			</cly-header>
-			<data-table @edit-user="onEditUser" :rows="users"></data-table>
-			<drawer :user="user" :features="features" :settings="drawerSettings" :controls="drawers.user"></drawer>
+			<data-table @refresh-table="refresh" @edit-user="onEditUser" :rows="users"></data-table>
+			<drawer @refresh-table="refresh" :user="user" :features="features" :settings="drawerSettings" :controls="drawers.user"></drawer>
 		</el-tab-pane>
-		<el-tab-pane class="tab-content" name="groups" lazy>
-			<span slot="label">User Groups</span>
-			<cly-header :title="i18n('management-users.groups-view-title')">
-					<template v-slot:header-right>
-							<div class="bu-level-item">
-									<el-button type="success" size="small" icon="el-icon-circle-plus">New User Group
-									</el-button>
-							</div>
-					</template>
-			</cly-header>
-			<data-table :rows="users"></data-table>
+		<el-tab-pane
```

Comment:
well well well,
i think you are still using the deprecated tabs.
we should be using `cly-dynamic-tabs`